Frontend/ui-driver/src/Flow.purs

```diff
@@ -241,6 +242,11 @@ loginFlow = do
   runInternetCondition
   void $ pure $ setCleverTapUserProp [{key : "Preferred Language", value : unsafeToForeign $ getValueFromConfig "defaultLanguage"}]
   setValueToLocalStore LANGUAGE_KEY $ getValueFromConfig "defaultLanguage"
+  if (getToBeLog unit) then do
```

Comment:
this is not required

Reply:
we can have two flags in purs global state, one will tell should we send the log and another flag for should we add timestamp in object if we have not sended the log

Reply:
We are sending the log to CleverTap only after we are getting the timestamp, adding the timestamp or not is being covered inside the `tobelog` check in the flow file at every place required. So why do we need 2 checks if we want to log just after we get the timestamp?

Reply:
@ashkriti how we will deal with cases if we are supposed to go to some other flow where this check is added and then to homescreen then both the timestamp is required, but only one will work out here